test/utils/CommonActions.js

```diff
@@ -145,48 +126,6 @@ class CommonActions {
         return this.getElement(elementCss).getText().toString() === textToVerify;
     }
 
-    /**
-     * @param elementCss auto completer textField.
-     * @param textToSelect select text on autoCompleter.
-     * @param autoCompleterKey locator to auto completer button on classic theme.
-     */
-    static selectOnAutoCompleterTextField(elementCss, textToSelect, autoCompleterKey) {
-        this.setInputTextField(elementCss, textToSelect);
-        if (isClassic) {
-            this.clickWebElement(autoCompleterKey);
-            this.autoCompleterClassicTheme(textToSelect);
-        }
-        else {
-            this.autoCompleterLightTheme(textToSelect, autoCompleterKey);
-        }
-
-    }
-
-
-    /**
-     * @param textToSelect select text on autoCompleter.
-     */
-    static autoCompleterClassicTheme(textToSelect) {
-
-
-        let handle = browser.getTabIds();
-        browser.switchTab(handle[1]);
-        browser.frame("resultsFrame");
-        let css = format('//a[text()="{}"]', textToSelect);
-        browser.pause(3000);
-        this.clickWebElement(css);
-        browser.switchTab(handle[0]);
-    }
-
-    /**
-     * @param textToSelect select text on autoCompleter.
-     * NEED IMPROVE
-     */
-    static autoCompleterLightTheme(textToSelect, autoCompleterKey) {
```

Comment:
Are you sure? this methods is to AutoCompleter Field on Classic and Lightning theme, maybe you need think this.

Reply:
To run in lightning and classic I decided to remove some, but I will add again for classic